python/test/unit/fem/test_assemble_cppimport.py

```diff
@@ -10,8 +10,8 @@
 from mpi4py import MPI
 
 import cppimport
```

Comment:
Should we just delete this test? It currently adds a dependency (test) on cppimport, even though we are skipping...

Reply:
I agree.